src/util/tests/encodeUrl.test.ts

```diff
@@ -18,7 +18,7 @@ import { stringToU8a } from '@polkadot/util';
 import { base64Encode } from '@polkadot/util-crypto';
 import { zlibSync } from 'fflate';
 
-import customTypesRialto from '../../configs/customTypesRialto.json';
+import customTypesRialto from '../../configs/substrateCustomTypes/customTypesChain1.json';
```

Comment:
This might break the test in case different env variables are set. I'd suggest to actually fetching the rialto types to make sure it's deterministic.